docs/source/whats_new.rst

```diff
@@ -10,16 +10,22 @@ For a full list of included Pull Requests and closed Issues, please see the
 Features
 --------
 
-* Kevin Donkers and Phil Elson made the AdaptiveScalar to the Natural
-  Earth Features the default. This will make the default features look
-  much nicer when plotting on zoomed in axes. (:pull:`1105`)
+* We are very pleased to announce that Greg Lucas has been added to the cartopy
+  core development team. Greg (@greglucas) added the NightShade feature in the
```

Comment:
I'm now a 'maintainer', you're on the core development team I think. I'm not sure the qualifiers on everything though. But, perhaps put your new upgrade in as well?

Reply:
I just copied the existing ones; it said the same thing when I was added. I don't think there are any announcements for maintainer->core dev I could copy.